doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -230,12 +231,12 @@ Please note that "shared" is a reference to multi-tenant support and pooled stor
 | Ref # | sub-category | Description |
 |---|---|---|
 | `req.kcm.07` | General | The Architecture **must** support policy driven horizontal auto-scaling of Kubernetes cluster. |
-
+| `req.kcm.04` | General | The Architecture **must** include Kubernetes artefacts (e.g., images, Helm charts, etc.) repository capabilities. |
```

Comment:
Not sure about this - need to understand which RM requirements this addresses.  As a more generic requirement for Kubernetes architecture, absolutely. But in the context of an architecture that is about standardising on the infrastructure capabilities consumed by workloads, I'm less certain.

Reply:
I actually agree with you here that I'm not sure if it should be in the RA. I included it because of: 
#610 
https://cntt-n.github.io/CNTT/doc/ref_arch/kubernetes/chapters/chapter01.html Figure 1-1 here
https://cntt-n.github.io/CNTT/doc/ref_arch/kubernetes/chapters/chapter03.html here 3.2.4
https://cntt-n.github.io/CNTT/doc/ref_arch/kubernetes/chapters/chapter04.html Figure 4-1

Maybe it would be better to adopt the language from 4.8? "The reference architecture must support the usage of a Kubernetes Application package manager using the Kubernetes API-s, like Helm v3."

Reply:
Yes I like that. And probably a new issue to clarify and make the documentation consistent around artefact repos.

Reply:
ok, let me create the new issue then split this PR out to make it a little more clear and separate the different issues